@ecomplus/storefront-components/src/scss/TheProduct.scss

```
@@ -130,6 +130,27 @@
     }
   }
 
+  .sale-timer {
+    width: 100%;
+    max-width: 300px;
+    background-color: var(--primary);
```

```suggestion
    background-color: var(--secondary-light);
```

Especialmente para não ser confundido com o botão comprar que é o principal CTA aí..

---

ok